### DISCUSSION TITLE
Thrustmaster Airbus Throttle Quadrant - happy how the A32NX reverse works after landing:

### DISCUSSION THREAD

**Discussion by Mickepilot** [flybywiresim/aircraft#3065] [Q&A] [answered]
Using the A32NX "Unstable Master Branch Build (with custom FBW)" I finally found the hidden "ThrottleConfiguration.ini" file in my PC under:
"C:\Users\ -the PC owner- \AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages\a32nx\work"

After some tweeks, I am now happy with the Thrustmaster Airbus TCA Quadrant, which now works correctly with detents matching on the throttle quadrant and the simulator. Please try the following steps:
 
1) In MS FS, reset the Thrustmaster A320 TCA Quadrant control sensitivity to "0" values (=linear response)
2) Do a new calibration of the TCA Quadrant, pressing both L & R red side buttons in all 5 detent positions (incl. full reverse position!)
3) Use values in the "ThrottleConfiguration.ini" file, that will get TCA Quadrant detents synchronized with the Flight Sim detents.

UPDATE 15th Februar 2021: After some more fine adjustments of the "ThrottleConfiguration.ini" values, I got my Thrustmaster Airbus TCA Quadrant to work even more precise synchronized with the throttle in the actual A320 MS FS (when using the A32NX custom FBW).
I now use these values:

Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.0
DetentReverseFull = -1.00
DetentReverseIdle = -0.70
DetentIdle = -0.44
DetentClimb = -0.10
DetentFlexMct = 0.53
DetentTakeOffGoAround = 1.00

Maybe manufacturing tolerances and calibration differencies could result that some sim folks needs other values for a perfect sync between the Thrustmaster Quadrant and the flight simulator.

**Comment by ChruutvoLuzi** [comment]
What Version are you using?

**Reply by Mickepilot** [reply]
The Unstable Master Branch Build (with custom FBW).

**Reply by ChruutvoLuzi** [reply]
> 
> 
> The Unstable Master Branch Build (with custom FBW).

i think you can use the key F2 for reverse, otherwise you need to edit the "ThrottleConfiguration.ini" file

there are some examples here:
https://github.com/flybywiresim/a32nx/blob/fbw/docs/README.md

**Reply by Mickepilot** [reply]
@ChruutvoLuzi, Hello, I am happy with the settings that I am using now, according to my first message, see point 3!
I do not need or use the F2 button, because I am using the Thrustmaster Airbus Throttle Quadrant!
The values in the link you provided can be tweeked, this works better for me: DetentReverseIdle = -0.70 & DetentClimb = 0.00.

**Reply by ChruutvoLuzi** [reply]
Ah, i thought you were asking how to do it, but looks like i misunderstood your question/information

**Reply by Tezbedz** [reply]
Hi Mickepilot
Could you explain the following:
reset the MS FS A320 TCA Quadrant control sensitivity to "0" values (=linear response)
Where do I do this?
I am OK with calibrating the Quadrant and once I find the throttleconfiuration.ini I can edite that.
Hopefully this will sort out the discrepancies in the detente

Terry Beddoes

**Comment by Mickepilot** [comment, marked as answer]
Hello Terry,
Ok, there is two things that must work together:
1) the Microsoft Flight Simulator
AND
2) the FBW "Unstable Master Branch Build (with custom FBW)".

You first have to reset control sensitivity to "0" by starting the MS FS.
Then go into "Options/Controls/TCA Q-ENG 1&2" Click on the blue field
called "Sensitivity". Now you can reset all axis. You should see stright
diagonal sensitivity lines, from left low corner, up to the right high corner.
Click "Done" and also "Apply & Save".
Once you found the "ThrottleConfiguration.ini" file (it's hidden) you can
now tweek the detents for the Quadrant.
(See my first comment on top)

Am Di., 2. Feb. 2021 um 18:46 Uhr schrieb Tezbedz <notifications@github.com

**Reply by Tezbedz** [reply]
Mickepilot

> Hello Terry, Ok, there is two things that must work together: 1) the Microsoft Flight Sim, AND 2) the FBW "Unstable Master Branch Build (with custom FBW)". You first have to reset control sensitivity to "0" by starting the MS FS. Then go into "Options/Controls/TCA Q-ENG 1&2" Click on the blue field called "Sensitivity". Now you can reset all axis. You should see stright diagonal sensitivity lines, from left low corner, up to the right high corner. Click "Done" and also "Apply & Save". Once you found the "ThrottleConfiguration.ini" file (it's hidden) you can now tweek the detents for the Quadrant. I did some adjustments last night, maybe it works for you as well: Log = true Enabled = true ReverseOnAxis = true ReverseIdle = true DetentDeadZone = 2.0 DetentReverseFull = -1.00 DetentReverseIdle = -0.70 DetentIdle = -0.44 DetentClimb = -0.10 DetentFlexMct = 0.53 DetentTakeOffGoAround = 1.00 Save the changes, and start the Sim again. You have to be on ground to check if the Thrustmaster Airbus Throttle detenst are correct. Have a nice evening, kind regards, Michael O. Am Di., 2. Feb. 2021 um 18:46 Uhr schrieb Tezbedz <notifications@github.com.

Thanks Michael
So to reset Control Sensitivity to "0" , all I need fo is turn on MSFS.
I just use the ordinary Dev build, but, I will follow the instructions and see what happens.

Do updates to the Aircraft affect the throttle.ini file?

Regards
Terry

**Comment by Mickepilot** [comment]
Sorry, if it's hard to understand... try it slowly.....
The "Sensitivity" is controlled in the Microsoft Flight Simulator! As I
explained before.
The correct "Detents" poistion in the Microsoft Flight Simulator is
controlled by the values (that I mentioned before) in the FBW Custom
"ThrottleConfiguration.ini" file for the A32NX.


Am Di., 2. Feb. 2021 um 19:41 Uhr schrieb Tezbedz <notifications@github.com

**Reply by Tezbedz** [reply]
Thanks, I sorted it out the other day and it worked beautifully,
Ive now updated the A320 and the throttle does not work again, I was wondering if I have to change the Throttle.ini each time I upload a new version of the A320NX, but, the throttle.ini file in the Works Folder was the same one I had saved for that very reason.
Do you have any ideas why it may stop working and thanks for your help last week.
Regards
Terry

**Reply by Tezbedz** [reply]
Sorry I should have said that now when the Thrustmaster Throttle is at Idle the A320 throttle is in Reverse Detent

**Reply by Tezbedz** [reply]
SORTED. Completely my fault I downloaded the wrong version of FBW A320

**Reply by CptBayer** [reply]
Hi Simmers,

today I got my whole set, Quadrant, Quadrant Addon and Joystick. I followed all this steps. But I have to Problems right now:

1. The Ini is not in the work folder (hidden files are set to show). Why is there no ini file?
2. The mapping for the spoiler and flaps lever is not working any more. Prior calibration it worked wonderfull. How to get this fixed? I unistalled the driver, deleted all USB advices which were not in use. But this didn´t solve the problem.

Thanx Klaus

**Comment by Mickepilot** [comment]
Hello CptBayer!
I don't have the Thrustmaster Quadrant Spoiler/Flaps Add-On yet (should get
them mid of April (!) according my dealer in Switzerland)
So I can only respond to your first question:
You have to use the A32NX unstable "Custom FBW" software Add-On in order to
find the "ThrottleConfiguration.ini" file.
You can see the values I use, in my first posting.
Kind regards, Michael

Am Mo., 15. Feb. 2021 um 19:56 Uhr schrieb CptBayer <
notifications@github.com>:

**Reply by CptBayer** [reply]
Hi Michael,

too bad, I tried the developer and the custom version. No ini file at all.... I am confused .... Any other hints? 

Greets Klaus

**Reply by CptBayer** [reply]
Ok. Corrction. Got It. Ini file is now on his place. All the values are working perfect! :-) .... Now the flap and spoiler lever are the next thing to gei it fixed .... 

Greets Klaus

**Reply by CptBayer** [reply]
In the flightsimlabs formum the solution for the calibration of all 4 levers together ... rock´n roll .... Thanx Abraham...

 Insert Pin provided by Thrustmaster to link both throttles.

Throttles, Speed Brakes and Flaps to full back. Make sure Gear Switch is in the Gear Down position.

Unplug the TQ fto PC cable at the back and press and hold the Black button under the Eng 2 switch, while at the same time, plugging in the controller to PC cable at the back again.

Now move ALL the levers to full forward position. i.e RET / TOGA / 0 position.

Press two red buttons on Throttles to set the TOGA position, and then flick the Gear to Up and then to Down to set the Speed Brakes RET and Flaps 0 position.

Move Throttle levers back to MAX FLX and Speed Brake to 1/4 and Flaps to 1 Press. Now press the red buttons to set Throttles and then Gear Up and down to set Speed Brakes and Throttles.

Move Throttle levers back to CL and Speed Brake to 1/2 and Flaps to 2. Now press red buttons to set Throttles and then Gear Up and down to set Speed Brakes and Throttles.

Move Throttle levers back to IDLE and Speed Brake back to 3/4 and Flaps back to 3 . Now press the red buttons to set Throttles and then Gear Up and down to set Speed Brakes and Throttles.

Move Throttle levers to MAX RVS and Speed Brake to FULL and Flaps to 4. Now press the red buttons to set Throttles and then Gear Up and down to set Speed Brakes and Throttles.

Finally press Both Black Buttons under the ENG switches at same time to finish calibrating.

Remove the Throttles linking pin, unplug TQ and re-plug it in.

(I had to tick the Reverse box in the Speed Brakes and Flaps TCA Q-Eng Axis Assignments in P3D) .

Now the Speed Brake, Throttle and Flaps Detents work.

This process ensures the calibration captures the positions of the spoilers and the flaps lever as well as the throttles. Otherwise, if you’re only doing the calibration on the throttle quadrant without  moving the spoilers and flaps in parallel, the TQ/PC will think that the spoilers and flaps are fully calibrated for their full range, while actually being set just on the fully retract position. 

Greets Klaus

**Reply by Yosemitebear** [reply]
> Throttles, Speed Brakes and Flaps to full back. Make sure Gear Switch is in the Gear Down position.

THX for sharing and if u don't want to tick the Reverse box, u can set the Speed Brakes to RET and Flaps to 0 at this step.

**Reply by Yosemitebear** [reply]
> Now move ALL the levers to full forward position. i.e RET / TOGA / 0 position.

And at this step move Speed Brakes and FLAPS to full

**Comment by CptBayer** [comment]
> 
> 
> Using the A32NX "Unstable Master Branch Build (with custom FBW)" I finally found the hidden "ThrottleConfiguration.ini" file in my PC under:
> "C:\Users\ -the PC owner- \AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages\a32nx\work"
> 
> After some tweeks, I am now happy with the Thrustmaster Airbus TCA Quadrant, which now works correctly with detents matching on the throttle quadrant and the simulator. Please try the following steps:
> 
>     1. In MS FS, reset the Thrustmaster A320 TCA Quadrant control sensitivity to "0" values (=linear response)
> 
>     2. Do a new calibration of the TCA Quadrant, pressing both L & R red side buttons in all 5 detent positions (incl. full reverse position!)
> 
>     3. Use values in the "ThrottleConfiguration.ini" file, that will get TCA Quadrant detents synchronized with the Flight Sim detents.
> 
> 
> UPDATE 15th Februar 2021: After some more fine adjustments of the "ThrottleConfiguration.ini" values, I got my Thrustmaster Airbus TCA Quadrant to work even more precise synchronized with the throttle in the actual A320 MS FS (when using the A32NX custom FBW).
> I now use these values:
> 
> Log = true
> Enabled = true
> ReverseOnAxis = true
> ReverseIdle = true
> DetentDeadZone = 2.0
> DetentReverseFull = -1.00
> DetentReverseIdle = -0.70
> DetentIdle = -0.44
> DetentClimb = -0.10
> DetentFlexMct = 0.53
> DetentTakeOffGoAround = 1.00
> 
> Maybe manufacturing tolerances and calibration differencies could result that some sim folks needs other values for a perfect sync between the Thrustmaster Quadrant and the flight simulator.

Hi, 

is it possible that after an update (development mode), all the detents are not working anylonger? Yesterday it work, today after the update it won´t... hmmm ... or do I have to repeat the calibration thing everytime after an update?

Greets Klaus ...

**Comment by Mickepilot** [comment]
Hi, is it possible that after an update (development mode), all the detents
are not working anylonger? Yesterday it work, today after the update it
won´t... hmmm ... or do I have to repeat the calibration thing everytime
after an update?

Greets Klaus ...

Hi Klaus, you are probably using the wrong A32NX version in the MS FS
A320neo:
Instead of the "Development" version, you must use the "Custom FBW" version.
The "ThrottleConfiguration.ini" values should NOT change with each "Custom
FBW" update.
Kind regards, Michael

Am Di., 16. Feb. 2021 um 19:38 Uhr schrieb CptBayer <
notifications@github.com>:

**Reply by CptBayer** [reply]
Hi Michael,

ah ok !!! I switched from Custom to Developer ... I´LL try after the update ......

Greets Klaus

**Reply by CptBayer** [reply]
Hi, 

it is working again :-)... One thing, when the lever is in CLB position, PFD shows thrust idle :-p .... 

Greets Klaus

**Comment by Yosemitebear** [comment]
Your config is so accurate for the latest Custom FBW! Thank you so much bro!

**Reply by Paytheon** [reply]
Works GREAT!
